examples/music-bot/src/music/subscription.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { Track } from './track';
 
 function wait(time: number) {
-	return new Promise((resolve) => setTimeout(resolve, time).unref());
+	return new Promise((resolve) => setTimeout(resolve, time, 'Success').unref());
```

Comment:
Could you replace this entire function with something like:

```ts
const { promisify } = require('util');
const wait = promisify(setTimeout);
```

Don't worry about the unref